inst/extdata/data/translate_mime_types.csv

```diff
@@ -45,9 +49,11 @@ import,application/vnd.google-apps.document,image/png,NA
 import,application/vnd.google-apps.document,image/x-bmp,NA
 import,application/vnd.google-apps.document,image/x-png,NA
 import,application/vnd.google-apps.document,text/html,NA
+import,application/vnd.google-apps.document,text/markdown,NA
 import,application/vnd.google-apps.document,text/plain,NA
 import,application/vnd.google-apps.document,text/richtext,NA
 import,application/vnd.google-apps.document,text/rtf,NA
+import,application/vnd.google-apps.document,text/x-markdown,NA
```

Comment:
`text/x-markdown` appears to be a rather weird/legacy MIME type, but if it's one of the official ones (and it is), I agree we need to plan for it. This is a note to myself.